frontend/screens/SearchScreen.tsx

```diff
@@ -85,75 +87,81 @@ function SearchScreen() {
   };
 
   return (
-    <View style={styles.searchScreen}>
-      <SearchPanel isFetchingData={isFetching} hasData={!!data} />
-      <Text>Number of posts: {queryInfo?.length}</Text>
-      <Text>Number of fetched: {data?.pages.flat().length}</Text>
-      <Text>Dog breeds to fetch: {filter.dogBreeds.join(', ')}</Text>
-
-      {filter.dogBreeds.length > 0 && !!data ? (
-        displayType !== 'Map' ? (
-          <FlashList
-            ref={searchRef}
-            numColumns={displayType === 'Grid' ? 2 : 1}
-            refreshing={isFetching}
-            onRefresh={() => refetchPosts()}
-            data={data?.pages.flat()}
-            keyExtractor={(item) => item._id}
-            renderItem={({ item }) => (
-              <GenericCard cardType={displayTypeDict[displayType]} key={item._id} post={item} />
-            )}
-            onEndReached={() => fetchNextPage()}
-            nestedScrollEnabled={true}
-            onEndReachedThreshold={0.2}
-            contentContainerStyle={styles.searchResultsList}
-            estimatedItemSize={400}
+    <>
+      <View style={styles.searchScreen}>
+        <View style={[defaultStyles.defaultPaddingHorizontal, { zIndex: 100, paddingBottom: 8 }]}>
+          <SearchPanel isFetchingData={isFetching} hasData={!!data} />
+          {data && <Text style={textStyles.h6}>Showing {data?.pages.flat().length} results</Text>}
+          <LinearGradient
+            colors={[theme.palette.background.primary, 'rgba(255,255,255, 0)']}
+            style={styles.bottomGradient}
           />
+        </View>
+        {filter.dogBreeds.length > 0 && !!data ? (
+          displayType !== 'Map' ? (
+            <FlashList
+              ref={searchRef}
+              numColumns={displayType === 'Grid' ? 2 : 1}
+              refreshing={isFetching}
+              onRefresh={() => refetchPosts()}
```

Comment:
Unnecessary anonymous function

```suggestion
              onRefresh={refetchPosts}
```